### PR TITLE
[Ready] New Mute quirk as well as Paper bin and crayon box to load-outs! - Requested by Suru

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
New quick Mute to gain three points
Two new Load Out items!
Crayon box
Paper bin
[why]
Was requested by Suru. 
Mute trait for people to be fake mimes I guess?
Crayon box for people that wish to paint or have a snack! - **Note people can use for food**
Paper bin for people to be able to have on hand everything you need to right letters to eacher, or throw paper airplanes at people - **Note Blood cult can benefit form this as well as people making paper sacks**

### CODE REVIEW

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -268,3 +268,16 @@
 	var/mob/living/carbon/human/H = quirk_holder
 	phobia = new
 	H.gain_trauma(phobia, TRAUMA_RESILIENCE_SURGERY)
+
+/datum/quirk/mute
+	name = "Mute"
+	desc = "Do to some accident caused you or by choice you have no voice to be heard!"
+	value = -3 //HALP MAINTS
```
Comment (deathride58) [inline code/datums/traits/negative.dm]: -3 is pretty major for something that's circumvented so easily

Comment (Trilbyspaceclone) [reply code/datums/traits/negative.dm]: Be a antag like blood cult or wiz and say that ^_^

Comment (Ragolution) [reply code/datums/traits/negative.dm]: Just to remind that we have Pacifist on... -2, I think? Even if it's -3, I hardly think being mute is worse than being a pacifist, or even on-par with it.

Comment (Trilbyspaceclone) [reply code/datums/traits/negative.dm]: Peacefull if antaged gets removed

**Review by silicons — CHANGES_REQUESTED** [review]
-2.

**Comment by silicons** [comment]
not -3.

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -268,3 +268,21 @@
...
+	medical_record_text = "Patient has an been unable to talk or use there voice."
+	var/datum/brain_trauma/severe/mute
+
+/datum/quirk/mute/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.gain_trauma(mute, TRAUMA_RESILIENCE_SURGERY)
+
+/datum/quirk/mute/on_process()
+	if(quirk_holder.mind && LAZYLEN(quirk_holder.mind.antag_datums))
+		to_chat(quirk_holder, "<span class='boldannounce'>Your antagonistic nature has caused your voice to be heard.</span>")
```
Comment (silicons) [inline code/datums/traits/negative.dm:287]: this is terrible but okay

**Review by silicons — APPROVED** [review]

File: code/datums/traits/negative.dm (hunk 2 above)
Comment (Ragolution) [reply code/datums/traits/negative.dm:287]: This has tons of spelling mistakes.

**Pull request merged**